src/codewhisperer/service/securityIssueHoverProvider.ts

```diff
@@ -41,6 +41,38 @@ export class SecurityIssueHoverProvider implements vscode.HoverProvider {
         return new vscode.Hover(contents)
     }
 
+    public updateRanges(event: vscode.TextDocumentChangeEvent) {
```

Comment:
This is a case where a docstring is desperately needed. What is the purpose of this function?
```suggestion
    /** .... */
    public updateRanges(event: vscode.TextDocumentChangeEvent) {
```

Reply:
Sure, will add a docstring here. The purpose of this function is to offset the hover positions when the document changes, basically the same thing for [diagnostics](https://github.com/aws/aws-toolkit-vscode/blob/d2f03d52a22109bf6b4c71d04581be171b59a5e3/src/codewhisperer/service/diagnosticsProvider.ts#L83-L88).